lib/seed_helper.rb

```diff
@@ -138,6 +139,9 @@ def import_raw_template(contents, vendor = 'Foreman')
     def import_templates(template_paths, vendor = 'Foreman')
       template_paths.each do |path|
         import_raw_template(File.read(path), vendor)
+      rescue RuntimeError => e
+        logger.error("ERROR #{e} parsing #{path.inspect}")
```

Comment:
Foreman::Logging.exception("Error in seededing the template metadata #{e.message}", exception)